openslides/agenda/static/templates/agenda/item-detail.html

```diff
@@ -37,6 +37,12 @@
 <div ng-if="item" class="details listOfSpeakers">
   <div class="pull-right">
     <span os-perms="agenda.can_manage">
+      <button class="btn btn-danger"
+          ng-bootbox-confirm="Are you sure to delete the list of all speakers?"
```

Comment:
Use better confirm question: "Are your sure you want to remove all speakers of this list?"